lightsoff_api/__init__.py

```diff
@@ -178,5 +180,36 @@ def create_place_review(path: PlacePath, body: PlaceReviewBody):
     }, HTTPStatus.OK
 
 
+class PlaceContactResponse(BaseModel):
+    code: int = Field(0, description="Status Code")
+    message: str = Field("ok", description="Exception Information")
+
+
+@app.post(
+    "/places/<string:google_place_id>/contact", responses={"200": PlaceContactResponse}
+)
+def update_place_count_of_contact(path: PlacePath):
```

Comment:
`def increase_place_contacted_count`?

Same for the endpoint `"/places/<string:google_place_id>/increase_contacted_count"`